st2common/bin/bootstraprmq.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python2.7
+
+import sys
+import st2common.transport.bootstrap as transport_bootstrap
+
+if __name__ == '__main__':
```

Comment:
Yes. Ops are idempotent but that does not mean we repeat them knowingly. 

Not wanting to use launchdev or st2_deploy is a very narrow usecase. Besides by this approach it needs to have run once before is all. 

You seem to have strong opinions against this approach so I shall change it around to your preference. I actually prefer this explicit way to doing things.